### DISCUSSION TITLE
Container unhealthy on a subdomain

### DISCUSSION THREAD

**Discussion by LeVraiRoiDHyrule** [qdm12/ddns-updater#577] [Q&A]
Hi, 

I am updating my dns on Cloudflare using this tool. I want to update the base A and AAAA records, so I've set "host": "@". But after that, the container is unhealthy. It looks like it is because I host the web UI on a subdomain : ddns.mydomain.com. I think this is the cause because "host": "ddns", the the container is healthy. But the problem is that now it only updates the record for the subdomain "ddns.mydomain.com", I want to update "mydomain.com".

So what would be my way out of this ? How to have the DNS lookup work when the UI is on a subdomain while I want to update the root domain ?

Thanks in advance for any answer, have a great day

**Comment by LeVraiRoiDHyrule** [comment]
Hi, would you have an idea of how I could fix this ?

**Comment by qdm12** [comment]
Hi there! Sorry for the 6monthsdelay; What unhealthy message do you get in your logs?
It shouldn't matter if you host the UI on ddns.mydomain.com and setting `"host": "@"` is the way to go to update the root domain.
